MagickCore/nt-base.c

```diff
@@ -1793,20 +1793,20 @@ MagickPrivate DIR *NTOpenDirectory(const char *path)
     {
```

Comment:
We should remove the null check here instead. `AcquireCriticalMemory` will never be null.

Reply:
if I will do it in new commit, can you squash all commits during merge ?

Reply:
I can squash your changes.